Lib/test/test_getargs2.py

```diff
@@ -626,16 +626,16 @@ def test_required_args(self):
             )
         # required arg missing
         with self.assertRaisesRegex(TypeError,
-            r"Required argument 'required' \(pos 1\) not found"):
+            r"required argument for function 'required' \(pos 1\) not found"):
```

Comment:
This looks as the function has the name "required". May be omit "for function" if function name is not specified?